### ISSUE TITLE
[NN] Dropout

### ISSUE THREAD

**Issue by michelole** [bst-mug/n2c2#102]
https://deeplearning4j.org/docs/latest/deeplearning4j-troubleshooting-training#regularization suggests `dropout(0.5)` to reduce overfitting.
